architecture/index.md

```diff
@@ -19,7 +19,7 @@ AppStudio is a platform for building integrated software that streamlines, conso
 - Build artifacts once with enough data to determine suitability for releasing.
 - Build artifacts once that can be released to multiple locations, multiple use cases.
 - Specify builds and their dependencies entirely from git and not from transient state of the build system. Employ tools like [renovate](https://docs.renovatebot.com/) to manage dependency updates.
-- Build semantic reproducible artifacts. Any configuration which has the potential to affect the semantic functionality of a build should be source controlled and associated with the produced artifact (via the provenance, for example).
+- Build semantic reproducible artifacts. Any configuration which has the potential to affect the semantic functionality of a build should be recorded in the provenance and source controlled when.
```

Comment:
I think that understanding is consistent with the change here. It affords the opportunity for maintainers to keep track of their dependencies in their build (i.e. whenever possible) but it also accepts that this may not be possible in some situations. As long as the build-affecting parameters are captured in the provenance, then a reproduction _should_ be possible if needed.

(and thanks for catching my abruptly ending thought!)